appengine/standard_python37/django/requirements.txt

```diff
@@ -1,2 +1,2 @@
-Django==2.1.14 #This requirement must remain pinned at 2.1.14 until https://github.com/PyMySQL/PyMySQL/issues/790 is resolved
+Django==3.0.5 #This requirement must remain pinned at 2.1.14 until https://github.com/PyMySQL/PyMySQL/issues/790 is resolved
```

Comment:
Should the note about pinning at 2.1.14 be removed?

Reply:
Yeah, it definitely should. Removed